src/lib/output/renderer.ts

```diff
@@ -199,14 +199,7 @@ export class Renderer extends ChildableComponent<Application, RendererComponent>
                 }
             }
 
-            var filename = Path.join(path, 'theme.js');
```

Comment:
Could we bring this back. As I understand this breaks typedoc-markdown-theme. It has some code in theme.js and now it's being ignored.